docs-src/content/getting-started/cubic_docker.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Installing the cubicsvr Image
+weight: 5
+---
+
+The SDK is used to call an instance of cubicsvr using gRPC.  Cobalt distributes a docker image that contains the cubicsvr binary and model files.
+
+<!--more-->
+
+1. Contact Cobalt to get a link to the image file in AWS S3.  This link will expire in two weeks, so be sure to download the file to your own server.
+
+2. Download with the AWS CLI if you have it, or with curl:
+
+    ```bash
+    URL="the url sent by Cobalt"
+    IMAGE_NAME="name you want to give the file (should end with the same extension as the url, usually bz2)"
+    curl $URL -L -o $IMAGE_NAME
+    ```
+   
+2. Load the docker image: 
+
+    ```bash
+    docker load < $IMAGE_NAME
```

Comment:
We usually bzip2 our images.

Reply:
Yes, docker load can handle a bz2 file directly. 

From the [docker docs](https://docs.docker.com/engine/reference/commandline/load/):
> Load an image or repository from a tar archive (even if compressed with gzip, bzip2, or xz) from a file or STDIN. It restores both images and tags.

Reply:
Good to know!